cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -742,9 +748,13 @@ message NodeJobTask {
      */
     required ClusterJobType jobType = 1;
     /**
-     * Backup name
+     * Backup directory
+     */
+    optional string backupDir = 3;
+    /**
+     * Determine if this is the first node participating in job
      */
-    optional string backupName = 2;
+    optional bool firstNodeInJob = 4;
```

Comment:
That's required because:
- we need to do table truncate during restore;
- table truncate should be done only once;
Please see: 
- io.mesosphere.mesos.frameworks.cassandra.executor.BackupManager#restore 
- io.mesosphere.mesos.frameworks.cassandra.CassandraFrameworkProtos.NodeJobTask#getFirstNodeInJob